### ISSUE TITLE
Menubar click event not fired

### ISSUE THREAD

**Issue by maierfelix** [Treverix/custom-electron-titlebar#5]
When clicking a menubar item, the click event is not fired.
The problem is in [this](https://github.com/Treverix/custom-electron-titlebar/blob/master/src/menubar.ts#L570-L581) line, where the click event is not fired in case the menu has a submenu.

**Desktop:**
 - OS: Windows 10
 - Electron version: 11.0.0-nightly.20200818
 - Node Version: 12.18.3

**Comment by Treverix** [comment]
This looks like wanted behavior. Why do you need the click event in this case? From what I see on the code, we want

- if the the item is a submenu (a node), the sub tree should open
- the the item is a menu item (a leaf), then it should emit the click event (menu item clicked)

I think I need to understand use cases, where we need click events (or custom events?) being fired when the user clicks on a sub menu node. 

I think, if we start using 'empty submenu items' like menu items, then the design starts getting wrong.

**Comment by maierfelix** [comment]
In the electron [docs](https://www.electronjs.org/docs/api/menu-item) I see that I can assign a `click` event to the menu item. I expected this to be able to use like this:

````ts
  const menu = new Menu();
  const pluginMenu = new MenuItem({
    submenu: [],
    click: () => { console.log("This event is never fired"); }
  });
  menu.append(pluginMenu);
  this._getTitlebar().updateMenu(menu);
````

Doing this with sub menu items (i.e `Menu->MenuItem->MenuItem`) the click event is fired and working fine.
But for upper menu items (i.e. `Menu->MenuItem`), the click event is never fired for me.

**Comment by Treverix** [comment]
I see. So electron seems to fire click events for the submenu type item also and honetly, I see no reason to do that. Clicking on that should expand the submenu. If need to react to a user click, then it's a not a submenu type item but a normal one.

Anyway, I take action but only to be consistent with electron. I don't encourage anyone to abuse subitem type menu items. But maybe someone wants to count how many times a user opened a menu. There it could be useful.

**Comment by maierfelix** [comment]
What about the intention to show a popup dialog when a user clicks the upper menu item. Is this possible already or affected by this issue as well?

**Comment by Treverix** [comment]
If the upper menu item is not of type 'submenu', then it should fire. We need to investigate - and I didn't use menus in titlebars, to be honest but I'll figure out :)

**Comment by Treverix** [comment]
Fixed with version 4.1.4. Thanks for reporting the issue!
